po/tr.po

```diff
@@ -301,24 +301,24 @@ msgstr "geçersiz parametre"
 
 #: src/confighandlerexception.cpp:19
 msgid "too few parameters."
-msgstr "yetersiz parametre"
+msgstr "pek az parametre."
 
 #: src/confighandlerexception.cpp:21
 msgid "unknown command (bug)."
-msgstr "bilinmeyen komut (bug)"
+msgstr "bilinmeyen komut (hata)."
 
 #: src/confighandlerexception.cpp:23
 msgid "file couldn't be opened."
-msgstr "dosya açılamadı"
+msgstr "dosya açılamadı."
 
 #: src/confighandlerexception.cpp:25
 msgid "unknown error (bug)."
-msgstr "bilinmeyen hata (bug)"
+msgstr "bilinmeyen hata (hata)."
 
 #: src/configparser.cpp:104
 #, c-format
-msgid "%s line %u"
-msgstr "%s satır %u"
+msgid "%s %u. satır"
```

Comment:
Hey, don't change msgid, that's the string by which Newsboat *finds* a translation ;)